ExampleOutput/NoCommand.g.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System.CommandLine.Parsing;
```

Comment:
This file is generated when the normal generation is not done to provide IntelliSense. 

It can't merely reside in a library because you need one, or the other, but not both.